### DISCUSSION TITLE
Cinnamon moving on...

### DISCUSSION THREAD

**Discussion by cinnamon-msft** [microsoft/terminal#14249] [Announcements]
The time has come for me to step down from my role as product manager of Windows Terminal. I am not going far and I am still working to improve the developer experience on Windows.

I would like to sincerely thank all of you who have been a part of the Terminal journey and subsequently my own.

To the Windows Terminal team, thank you all for being just wonderful people to work with. I have learned so much from each and every one of you and I can truly call each one of you a friend. I started on this team right out of university and you were all so welcoming and patient with me, and I could not have asked for anything more.

To the Windows Terminal community, thank you for being a huge part of the Terminal journey and for working with us to make a terrific product. Being able to work with you has been the most rewarding part of being on this team and I hope to continue working with you as I too become part of the community.

I have just posted my final Terminal blog post and I will still be around to answer questions when possible. I am excited for the future of Windows Terminal and I will be following along every step of the way. This is not a goodbye, but rather a see you soon. ❤️

**Comment by dgurney** [comment]
Thanks for all your hard work!

**Comment by evilpro** [comment]
Thanks for everything!

**Comment by bplasmeijer** [comment]
Thank you, and the team.

**Comment by bitcrazed** [comment]
Yowzah! HUGEST thanks to you for all your patience, enthusiasm, and of course, all your hard work over the last few years, Kayla. Very best wishes for your future - can't wait to see what you get up to next 😜😁

**Comment by DHowett** [comment]
Kayla, it goes without saying that the Terminal team wouldn't be half where it is without you. Thank you. I'll miss you, but I am excited about what you've got in the pipeline! :smile:

**Comment by craigloewen-msft** [comment]
Kayla I'm so sad and so excited at the same time! I loved everything you've helped create with Windows Terminal and cannot wait to see the cool stuff you build next!

**Comment by PaulFWatts** [comment]
All the very best for your future career!

Live Long and Prosper :-)

**Comment by arsatali91** [comment]
Thanks for all the great and passionate works on the Windows Terminal, really amazed to witness the way it had evolved and become the favourite for many devs. All the best for your next journey, looking forward for what you got in pipeline!

**Comment by timsneath** [comment]
Congratulations: with Terminal now the default, you've successfully unravelled a giant ball of wax and turned it into a thing of beauty. Not many have been so successful at revitalizing a key component of Windows that is touched by billions of people; you have an incredible legacy already, and I can't wait to see what you fix next :)

**Comment by creizlein** [comment]
You will be certainly missed, but foremost thank you very very much for your work, time and efforts towards WT!
Looking forward to see your posts again in other projects, let us know where! 👏 👏 👏
